src/index.html

```diff
@@ -9,6 +9,81 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header-moyo">
+      <a class="logo-moyo" href="#home">
+        <img src="images/logo.png" alt="moyo-logo">
+      </a>
+      <nav class="nav-bar">
+        <ul class="nav__list">
+          <li class="link-item">
+            <a
+              href="#"
+              class="nav__link is-active"
+              >
+              Apple
+            </a>
+          </li>
+          <li class="link-item">
+            <a
+              href="#l"
+              class="nav__link"
+              >
+              Samsung
+            </a>
+          </li>
+          <li class="link-item">
+            <a
+              href="#"
+              class="nav__link"
+              >
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="link-item">
+            <a
+            data-qa="hover"
+            href="#"
+            class="nav__link nav__link_laptops"
```

Comment:
```suggestion
            class="nav__link"
```
Для чого потрібен клас nav__link_laptops?